entity-framework/core/querying/overview.md

```diff
@@ -40,4 +40,4 @@ The most common operations that result in the query being sent to the database a
 * Databinding the results of a query to a UI
 
 > [!WARNING]  
-> **Always validate user input:** While EF does provide protection from SQL injection attacks, it does not do any general validation of input. Therefore if values being passed to APIs, used in LINQ queries, assigned to entity properties, etc., come from an untrusted source then appropriate validation, per your application requirements, should be performed. This includes any user input used to dynamically construct queries. Even when using LINQ, if you are accepting user input to build expressions you need to make sure than only intended expressions can be constructed.
+> **Always validate user input:** While EF does provide protection from SQL injection attacks, it does not do any general validation of input. Therefore if values that are being passed to APIs, used in LINQ queries, assigned to entity properties etc., come from an untrusted source then appropriate validation, per your application requirements, should be performed. This includes any user input used to dynamically construct queries. Even when using LINQ, if you are accepting user input to build expressions, you need to make sure that only intended expressions can be constructed.
```

Comment:
Agreed that grammar can be improved, thanks for the PR, but looking at this, I think there is an opportunity for polishing and simplifying things even more. For examples, by removing "being" instead of adding "that are", and even getting rid of etcetera, because "passed to APIs" covers pretty much all the cases anyway. 

If we don't remove the etcetera, getting rid of the comma seems incorrect. 

And I think there should be a comma before "then".

Proposal (including some other smaller changes):

>**Always validate user input:** While EF Core does provide protection from SQL injection attacks, it does not do any general validation of input. Therefore if values passed to APIs, used in LINQ queries, or assigned to entity properties come from an untrusted source, then appropriate validation, per the application's requirements, should be performed. This includes any user input used to dynamically construct queries. Even when using LINQ, if you are accepting user input to build expressions, you need to make sure that only intended expressions can be constructed.

Reply:
Yeah, that statement is pretty long on the first place. Maybe we should shorten it, by removing examples or breaking them out.

Reply:
But the proposal could work for time, thanks.

Reply:
Thanks @keyurpd. I will merge your PR and the I will make the additional edits.